src/index.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash';
+import './style.css';
+import UpdateUI from './components/UpdateUI.js';
+import Todolist from './components/Todolist.js';
+import storageManager from './components/Storage.js';
+import Methods from './components/Methods.js';
+import { markCompleted, clearCompleted } from './components/Completed.js';
+
+const listText = document.querySelector('.input-task');
+const addListBtn = document.querySelector('#add');
+const listContainer = document.querySelector('.list');
```

Comment:
Because the list is also a Container,  I don't know if listContainer should be renamed by list

Reply:
😄 I have renamed it to `todosHolder`.